guides/micronaut-openapi-generator-server/java/src/main/java/example/micronaut/BookService.java

```diff
@@ -0,0 +1,32 @@
+package example.micronaut;
+
+import example.micronaut.model.BookAvailability;
+import example.micronaut.model.BookInfo;
+import jakarta.inject.Singleton;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Singleton // <1>
+public class BookService {
```

Comment:
I think we should save the `BookInfo` in a MySQL database with Flyway migrations and using Micronaut data JDBC. 

I think this will involve creating a `JdbcRepository` a `MappedEntity` and a flyway migration.

Reply:
Maybe we can add it as a next step (like I added some comments about security)?
I don't think it should be required to have MySQL database to complete the guide.